src/common/actions/repository.js

```diff
@@ -36,12 +36,13 @@ export function buildRepositories(repositories, owner) {
   };
 }
 
-export function buildRepository(repository) {
+// add a repository to launchpad's build queue, returns a snap representation
```

Comment:
the fetch might, but we're not handling it here are we? it's just updating the original repository state i think